doc/source/whatsnew/v1.5.0.rst

```diff
@@ -22,6 +22,29 @@ Styler
   - New method :meth:`.Styler.to_string` for alternative customisable output methods (:issue:`44502`)
   - Various bug fixes, see below.
 
+.. _whatsnew_150.enhancements.from_dummies:
+
+from_dummies
+^^^^^^^^^^^^
+
+Added new function :func:`~pandas.get_dummies` to convert a dummy coded :class:`DataFrame` into a categorical :class:`DataFrame`.
+
+Example::
+
+.. code-block:: python
+
+    >>> import pandas as pd
+
+    >>> df = pd.DataFrame({"col1_a": [1, 0, 1], "col1_b": [0, 1, 0],
+    ...                    "col2_a": [0, 1, 0], "col2_b": [1, 0, 0],
+    ...                    "col2_c": [0, 0, 1]})
+
+    >>> pd.from_dummies(df, sep="_")
+        col1    col2
+    0    a       b
+    1    b       a
+    2    a       c
```

Comment:
as this'll be a new feature, it should OK to make this an `ipython` block and it'll get rendered

hard-coded code-blocks are only really for when comparing with old behaviour (which wouldn't work when rendered)

Reply:
Okay will change that. Should I also add a link to the user guide entry for `from_dummies`? I wanted to get a first draft pushed to the the PR to shere where CI complains and then see where to extend it.

Reply:
I don't know if a link to the user guide is typically added for new features, perhaps check old whatsnew notes to see

> to see where the CI complains

😄  the docs take a long time to build locally, but if you just build a single page (e.g. the new whatsnew note) it's pretty fast, and that should show you the warnings, there's some examples [here](https://pandas.pydata.org/docs/development/contributing_documentation.html#building-the-documentation) with `--single`